src/components/Results.tsx

```diff
@@ -26,6 +26,18 @@ const Results: React.FC<{
       <button onClick={resetQuiz} className="results-btn">
         Play again?
       </button>
+
+      <a
+        className="results-text"
+        href={
+          "http://twitter.com/intent/tweet?text=I just scored " +
+          ((Math.floor(points) / totalPoints) * 100).toString() +
+          "% on the Learn to developerquiz.org - try it yourself?" +
+          "&hashtags=freecodecamp"
+        }
+      >
+        <i className="fab fa-twitter"></i>Tweet your quiz score
```

Comment:
small thing, but let's make this a self closing tag

```suggestion
        <i className="fab fa-twitter"/>Tweet your quiz score
```

Reply:
Yeah self-closing are better in this case.